analysis/ActivityTools.py

```diff
@@ -251,5 +251,19 @@ def DetectedCoincidences( DecayRates, DecayData, SimulationWindow, CoincidenceWi
     #    print( str( lostPhotons ) + " photons lost of " + str( totalPhotons ) + " total (" + str( lostPhotons * 100 / totalPhotons ) + "%) and " + str(recycledPhotons) + " recycled, " + str(windowsFromRecycled) + " recycled to new window" )
 
     y, x, patches = mpl.hist( hitRadii, bins=26, range=[-130, 130] )
+    #counts, bins 
     mpl.clf()
-    return NECRFromHistogram( x, y, SimulationWindow )
+    # return NECRFromHistogram( x, y, SimulationWindow )
+    # Hanna: First find the true and R+S coincidences
```

Comment:
No need to prefix comments with names, just makes things untidy. Either they are a useful explanation for everyone, or they are part of your debugging/development process and should be removed.

Reply:
Removed the name but left the comments as these are useful